data/pokemon.json

```diff
@@ -1139,6 +1139,11 @@
   },
   {
     "name": "ho-oh",
+    "nickname": [
+      "hooh",
+      "hoho",
+      "ho-ho"
```

Comment:
'ho-ho' actually isn't needed as an explicit nickname due to how the fuzzy lookup works (I was doing some testing) :)

Reply:
Cool; my cursory peek through didn't find any fuzzy matching, but I'm glad it's happening. I'll update that straight away.